src/Hedgehog/Report.fs

```diff
@@ -3,25 +3,28 @@ namespace Hedgehog
 [<Measure>] type tests
 [<Measure>] type discards
 [<Measure>] type shrinks
-
-[<RequireQualifiedAccess>]
-type RecheckType =
-    | None
-    | CSharp
-    | FSharp
-    
     
 [<Struct>]
 type RecheckData = internal {
     Size : Size
     Seed : Seed
 }
```

Comment:
Wild idea - `RecheckData` could hold `Language/RecheckType`...?

Reply:
No.  `RecheckData.serialize` is _the_ function that defines what `RecheckData` is, and it doesn't care about the type.